packages/playback-controls/src/playback-controls.ts

```diff
@@ -102,12 +108,21 @@ export default class PlaybackControls extends LitElement {
   }
 
   handleVolumeChange() {
-    if (this.volume === 1) {
-      this.volume = 0;
+    let newVolume = this.volume;
+
+    if (newVolume === 1) {
+      // we cannot use a volume of 0 since it becomes "falsey" and causes it to not be set
```

Comment:
So, you are starting at 1, what is max? 10?  What is mute, 1?

Reply:
what is the "thing" that reads it as falsey?

Reply:
The range is 0 to 1, but we're sort of shortcutting the `0` to `0.00001` so it doesn't become falsey.

Reply:
🤦‍♂ I just found the thing intercepting the falsey value and stopping. Going to update this MR.